src/scene.js

```diff
@@ -343,6 +338,58 @@ export default class Scene {
     return hash[[r,g,b]];
   }
 
+  pickCustom(x, y, opt = {}) {
+    const gl = this.gl;
+
+    if (this.pickingFBO === undefined) {
+      this.pickingFBO = new Framebuffer(gl, {
+        width: gl.canvas.width,
```

Comment:
Yes, I changed it to a simpler scissoring scheme, hence the buffer size change. There may be a way to do that with a small buffer; I can research it if it is an issue.